dockerfile/rocm4.0-pytorch1.7.0.dockerfile

```diff
@@ -72,8 +72,9 @@ ENV PATH="${PATH}" \
 
 WORKDIR ${SB_HOME}
 
+ARG ROCM_ARCH=gfx908
```

Comment:
why this image is only for MI100, can't it run for other gpus?

Reply:
can use docker build --build-arg ROCM_ARCH to change it, besides,  do you have a better choice to pass the ROCM_ARCH value?